plugins/destination/file/client/read.go

```diff
@@ -4,20 +4,28 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"strings"
+	"time"
 
-	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/apache/arrow/go/v12/arrow"
+	"github.com/cloudquery/plugin-sdk/v2/schema"
+	"github.com/google/uuid"
 )
 
-func (c *Client) Read(ctx context.Context, table *schema.Table, sourceName string, res chan<- []any) error {
+func (c *Client) Read(ctx context.Context, arrowSchema *arrow.Schema, sourceName string, res chan<- arrow.Record) error {
+	tableName := schema.TableName(arrowSchema)
 	if !c.pluginSpec.NoRotate {
-		return fmt.Errorf("reading is not supported when no_rotate is false. Table: %q; Source: %q", table.Name, sourceName)
+		return fmt.Errorf("reading is not supported when no_rotate is false. Table: %q; Source: %q", tableName, sourceName)
```

Comment:
```suggestion
		return fmt.Errorf("reading is not supported when `no_rotate` is false. Table: %q; Source: %q", tableName, sourceName)
```
More changes along the same lines as a general clean up?